pkg/apiserver/apiserver.go

```diff
@@ -277,6 +311,22 @@ func InstallLogsSupport(mux Mux) {
 	mux.Handle("/logs/", http.StripPrefix("/logs/", http.FileServer(http.Dir("/var/log/"))))
 }
 
+// Adds a service to return the supported api versions.
```

Comment:
It would be a little nicer to put AddApiWebService and APIVersionHandler (which could be changed to apiVersionHandler) next to each other, and handleVersion and InstallSupport (which could be renamed InstallVersion) next to each other, but that can be left to a future PR.